### DISCUSSION TITLE
Got marked spam

### DISCUSSION THREAD

**Discussion by Arasans** [community/community#61216] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i have got this message when i tried to connect my versel to github "You are marked as spam, and therefore cannot authorize a third party application."

**Comment by Belxz777** [comment]
i got this error message whenever i tried to push my code

github team resolve my issue after going through the following steps
To resolve the issue of being marked as spam on GitHub and not being able to authorize a third-party application, you can follow the steps.

Login to Your GitHub Account: Make sure you are logged into your GitHub account.

Enable Two-Factor Authentication (2FA): Before you can open a support ticket, you will need to enable Two-Factor Authentication (2FA) on your GitHub account if you haven't already. This is an extra layer of security for your account.

Contact GitHub Support: Scroll to the bottom of the GitHub page and look for the "Contact" link. This is usually found in the footer of the page.

Go to your GitHub account settings.
Click on "Security" in the left sidebar.
Scroll down to the "Two-factor authentication" section.
Follow the prompts to enable 2FA and choose your preferred method (e.g., SMS, authentication app, or recovery codes).
Open a Ticket: Once 2FA is enabled, you should be able to open a support ticket. In the "Contact" section, look for a way to "Submit a request" or "Open a support ticket." Provide a detailed description of the issue you're facing and kindly request their assistance in resolving the problem.

GitHub support will review your ticket and hopefully help you resolve the issue.

Remember to be polite and clear in your support ticket. The GitHub support team is generally responsive and will assist you in resolving the problem. Good luck with

**Comment by ettaboyle** [comment, marked as answer]
Hello, thank you for checking in. This account and topic has been flagged as: >One person or legal entity may maintain no more than one free Account.

Our private support team is the only one able to help with support requests on account flags. You will need to follow up with them directly through your open ticket.

**Reply by ghost** [reply]
I have the same issue but the GitHub support doesn't even answer me. It's been 2 weeks already.

**Reply by Titi68730** [reply]
It's been more than 2 years for me...

**Reply by mressid** [reply]
I have the same issue here (Hello!!)

**Comment by K-Sakthi-Vel** [comment]
Hello GitHub,
My GitHub account username (sakthivel-programmer) was logged by someone on May 16, 2024, they added their mail id (sacral_zionist978829@fthcapital.com) and removed my mail id (klsakthi333@gmail.com) this was my only GitHub account where i kept all my projects now its hacked by sacral_zionist. i tried login with my sakthivel-programmer account(klsakthi333@gmail.com) but it shows user not exist i so much worried about this cause, please help me recover my GitHub account :(

**Reply by SuchyYT** [reply]
I feel your pain. Something similar happened to me but with my Microsoft account.

**Reply by snooproxy** [reply]
fthcapital.com is a temporary mail address from mail.tm
speak with github support. since you have access to the original email address they will probably return your account.

**Comment by El3wady1** [comment]
my account are marked as spam, and therefore cannot authorize a third party application.  how can i fix??

**Comment by ladygalactica** [comment]
My account is marked as spammy,, please help
